src/python/pants/option/global_options.py

```diff
@@ -135,12 +133,6 @@ def register_bootstrap_options(cls, register):
              help='Enables use of the v2 engine.')
 
     # These facilitate configuring the native engine.
-    register('--native-engine-version', advanced=True,
-             default=pkg_resources.resource_string('pants.engine', 'native_engine_version').strip(),
-             help='Native engine version.')
-    register('--native-engine-supportdir', advanced=True, default='bin/native-engine',
```

Comment:
I suppose since these are largely obscured from users in general it's fine to yank these without a deprecation cycle.

Reply:
Yea, I'm fine with this.

Reply:
Agreed that it's easy enough to do, fixed.